secret_handlers.go

```diff
@@ -120,3 +120,38 @@ func SecretGet(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, secret.ToSecretResponse())
 }
+
+func SecretEdit(c echo.Context) error {
+	secretDao, err := getSecretDao(c)
+	if err != nil {
+		return err
+	}
+
+	updateRequest := &m.SecretEditRequest{}
+	err = c.Bind(updateRequest)
+	if err != nil {
+		return util.NewErrBadRequest(err)
+	}
+
+	paramID, err := util.InterfaceToInt64(c.Param("id"))
+	if err != nil {
+		return err
+	}
```

Comment:
I believe here you should return` util.NewErrBadRequest(err)` because it is a bad request when something wrong is provided as an id